controllers/storagecluster/storageclient.go

```diff
@@ -67,6 +75,14 @@ func (s *storageClient) ensureCreated(r *StorageClusterReconciler, storagecluste
 
 		controllerutil.AddFinalizer(storageClient, internalComponentFinalizer)
 
+		if storagecluster.Spec.Network != nil && storagecluster.Spec.Network.IsMultus() {
+			cephNWAnnotationValue, err := getCephNetworkAnnotationValue(storagecluster.Spec.Network, storagecluster.Namespace)
+			if err != nil || cephNWAnnotationValue == "" {
```

Comment:
the function should know to send an error if empty value is not valid and you should check only for err or not.